### PR TITLE
Docker fix

### PR DESCRIPTION
**Pull request by y4my4my4m**
This is [this PR ](https://github.com/NoCode-NoLife/melia/pull/256) but with the additional fix you mentioned and some others

### CODE REVIEW

File: sql/update-2024-08-11_2.sql [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+
+-- Drop the second foreign key if it exists
+-- ALTER TABLE `storage_team`
+-- DROP FOREIGN KEY `storage_team_ibfk_2`;
+
+-- Add the first foreign key constraint
 ALTER TABLE `storage_team`
-  ADD PRIMARY KEY (`accountId`,`itemId`),
-  ADD KEY `itemId` (`itemId`);
+ADD CONSTRAINT `storage_team_ibfk_1` FOREIGN KEY (`accountId`) REFERENCES `accounts` (`accountId`) ON DELETE CASCADE ON UPDATE CASCADE;
```
Comment (y4my4my4m) [inline sql/update-2024-08-11_2.sql:19]: ive modified these two SQL because i was getting errors about it having two primary keys btw

**Comment by y4my4my4m** [comment]
seems like i dont have access to resolve the conflicting files...

**Comment by exectails** [comment]
Are you certain it's an access issue? I'm getting the message that the conflicts are "too complex to resolve in the web editor", but you should be able to do it locally.

**Comment by y4my4my4m** [comment]
@exectails you're right, i'll work on them locally.

**Comment by y4my4my4m** [comment]
Those changes were pulled in the other fork. So this PR is the same as PR #256 now. Closing

**Pull request closed without merging**